rook/scripts/zap-disk-auto.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# THIS WILL ZAP DISKS WITHOUT REQUIRING CONFIRMATION - USE `zap-disk.sh` INSTEAD
+# DON'T RUN THIS
+
+set -e -o pipefail
+
+here="$(dirname "$(readlink -f "$0")")"
+
+if [[ "$#" -lt 1 || "$#" -gt 2 ]]; then
+    echo "${0} <host ip> [sdb]"
+    exit 1
+fi
+
+HOST_IP=${1}
+#Check that it is a valid IP
+if [[ ! (${HOST_IP} =~ ^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$) ]]; then
+  echo "ERROR: host ip ${HOST_IP} is not a IP address."
+  exit 1
+fi
+
+#Set disk
+if [[ "$#" -eq 1 ]]; then
+    echo -n "Enter disk to wipe (Continue for default: sdb):"
+    read -r disk
+    if [ -z "${disk}" ]; then
+        DISK=sdb
+    else
+        DISK="${disk}"
+    fi
+    echo ""
+else
+    DISK=${2}
+fi
+
+echo "WARNING!"
+echo "This script will wipe the disk /dev/${DISK} on machine ${HOST_IP}"
+
+echo "Staring to wipe disk"
+ssh -i "${CK8S_CONFIG_PATH}"/id_rsa  -oStrictHostKeyChecking=no ubuntu@"${HOST_IP}" DISK="/dev/${DISK}" 'bash -s' < "${here}/zap-disk"
```

Comment:
We don't store any private ssh keys here I think

```suggestion
ssh  -oStrictHostKeyChecking=no ubuntu@"${HOST_IP}" DISK="/dev/${DISK}" 'bash -s' < "${here}/zap-disk"
```

Reply:
Normally we don't, but it was needed in the pipeline, similar to https://github.com/elastisys/compliantkubernetes-apps/blob/main/.github/workflows/end-to-end.yml#L123

Reply:
Right I see